pkg/portal/cluster/clusteroperators.go

```diff
@@ -27,15 +36,25 @@ func clusterOperatorsInformationFromOperatorList(operators *configv1.ClusterOper
 	for _, co := range operators.Items {
 		var Available = configv1.ConditionUnknown
 
+		var conditions []OperatorCondition
 		for _, cnd := range co.Status.Conditions {
+			var condition OperatorCondition
 			if cnd.Type == "Available" {
 				Available = cnd.Status
 			}
+			condition.Message = cnd.Message
+			condition.Reason = cnd.Reason
+			condition.Status = cnd.Status
+			condition.LastUpdated = cnd.LastTransitionTime.String()
+			condition.Type = cnd.Type
```

Comment:
could we transform that into a struct instantiation? 
```suggestion
			if cnd.Type == "Available" {
				Available = cnd.Status
			}
			condition := OperatorCondition{
				Message: cnd.Message,
				....
			}
```

Reply:
@facchettos I accidentally merged these changes into this PR. All cluster opertor admin portal changes are on this PR https://github.com/Azure/ARO-RP/pull/2587. I will address these comments there

Reply:
haha no worries !